packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -50,6 +55,10 @@ typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q
 /// method should be used to execute parallel tasks that reduce the risk of
 /// skipping frames.
 ///
+/// [compute] internally uses [Isolate.run] and provides additional timing
```

Comment:
nit: sentences in docs should start with a capital letter. You may have to restructure this sentence a little bit so it doesn't start with lower-case `compute`.

Reply:
@dnfield recommended I just add a section to `See also` and skip documenting the internals all together

Reply:
That sounds good, too. Sorry I didn't see that comment.